packages/collection-model/lib/model.js

```diff
@@ -184,6 +183,34 @@ const CollectionModel = AmpersandModel.extend(debounceActions(['fetch']), {
         return getNamespaceInfo(this._id).normal;
       },
     },
+
+    isTimeSeries: {
+      deps: ['type'],
+      fn() {
+        return this.type === 'timeseries';
+      },
+    },
+
+    isView: {
+      deps: ['type'],
+      fn() {
+        return this.type === 'view';
+      },
+    },
+
+    sourceId: {
+      deps: ['view_on'],
+      fn() {
+        return this.view_on ? `${this.database}.${this.view_on}` : null;
```

Comment:
This is just keeping the compat with what we are using right now as a `source` value when new collection tab is created, changing this will probably touch most of the plugins in compass (at least all collection tab related ones) and I would want to avoid that